src/query/service/tests/it/tests/sessions.rs

```diff
@@ -77,6 +78,7 @@ pub struct TestGlobalServices {
     session_manager: Mutex<HashMap<String, Arc<SessionManager>>>,
     users_manager: Mutex<HashMap<String, Arc<UserApiProvider>>>,
     users_role_manager: Mutex<HashMap<String, Arc<RoleCacheManager>>>,
+    query_config: Mutex<HashMap<String, Config>>,
```

Comment:
Config a not small struct, It's better to wrap inside `Arc`. We will not change it after init it.